ruuvi_examples/ruuvi_firmware/main.c

```diff
@@ -282,97 +296,149 @@ ret_code_t lis2dh12_int2_handler(const ruuvi_standard_message_t message)
+                        nrf_delay_ms(1000u);  }
+ 
+
+  if (model_plus) GREEN_LED_ON;    // will be turned off in power_manage.
+
+  nrf_delay_ms(MAIN_LOOP_INTERVAL_RAW + 100); // Delay before advertising so we get valid data on first packet
+
+  bluetooth_advertising_start(); NRF_LOG_INFO("Advertising start  \r\n");           // can this fail ?
+
+  if (init_status) {     // Report on status of initalization 
+   url_buffer[9]=init_status>>8;url_buffer[10]=init_status & 0xFF;       //  URL ruu.vi/#xx
+   bluetooth_set_eddystone_url(url_buffer, sizeof(url_buffer) ); nrf_delay_ms(500u); // keep sending this for a while
+//  raspberry pi:  
+//  export bdaddr='zz yy xx ww vv uu' #  MAC address least significant byte FIRST
+//  hcidump --raw |g  --line-buffered --after-context=2 $bdaddr
+//  04 3E 2B 02 01 00 01 C2 24 68 3C 10 C7 1F 02 01 06 03 03 AA
+//                       zz yy xx ww vv uu
+//  FE 17 16 AA FE 10 F6 03 72 75 75 2E 76 69 2F 23 10 00 00 00
+//                          r  u  u  .  v  i  /  #  ^^^^^--init_status example BATTERY_FAILED_INIT
+
+  // like format 03  too
+   data_buffer[0]   =   0x13 ;                              // NOT SENSOR_TAG_DATA_FORMAT 03;  (but could be intreperted as such)
+   data_buffer[1]   =    254;                                                          // NOT data->humidity;
+   data_buffer[2]   =    125;                data_buffer[3]  =    125;                 // NOT data->temperature   
+   data_buffer[4]   =  11125        >>8;     data_buffer[5]  =   11125       & 0xFF;   // NOT data->pressure       
+   data_buffer[6]   =  0x1D1D       >>8;     data_buffer[7]  =   0x1D1D      & 0xFF;   // NOT accx              I did 
+   data_buffer[8]   =  0x0BAD       >>8;     data_buffer[9]  =   0x0BAD      & 0xFF;   // NOT accy                     bad
+   data_buffer[10]  = init_status   >>8;     data_buffer[11] =  init_status  & 0xFF;   // NOT accz                xx xx
+   data_buffer[12]  =    vbat       >>8;     data_buffer[13] =    vbat       & 0xFF;
+//  from raspberry pi  
+//      export bdaddr='zz yy xx ww vv uu' MAC address least Significant byte FIRST
+//      hcidump --raw |  grep --line-buffered --after-context=2  "$bdaddr"
+//      > 04 3E 2B 02 01 00 01 C2 24 68 3C 10 C7 1F 02 01 06 1B FF 99
+//                             zz yy xx ww vv uu
+//        04 13 FE 7D 7D 2B 75 1D 1D 0B AD 10 00 08 41 00 00 00 00 00
+//                        +  u I did   bad sssss vv vv   ssss is init status example BATTERY_FAILED_INIT; vv vv=vbat=0x0841 ie 2.113v
+  bluetooth_set_manufacturer_data(data_buffer, sizeof(data_buffer)); nrf_delay_ms(500u); // keep sending this for a while
+                     }            
+
+  // Enter main loop. spinning here interrupts will bring us out frequently
+  // accelerometer invokes lis2dh12_int2_handler
   for (;;)
   {
     app_sched_execute();
-    power_manage();
+    power_manage();         // calls sd_app_evt_wait
   }
 }
+
+
+
+
+/* vim: set columns=140: */

```

Comment:
vim specific comments should be removed

Reply:
vim comment causes the width of the screen to be 140. Very helpful.

Reply:
editor-specific settings/configurations should be kept out of source control, unless it's a common setting for the sources (ie. source formatting). editor-specific screen width sounds like a personal preference setting

Reply:
removed